bin/webpack-configs.js

```diff
@@ -367,7 +365,39 @@ const getFrontConfig = ( options = {} ) => {
 			concatenateModules:
 				isProduction && ! process.env.WP_BUNDLE_ANALYZER,
 			splitChunks: {
-				automaticNameDelimiter: '--',
+				cacheGroups: {
+					default: false,
+					vendors: {
+						test: /[\\/]node_modules[\\/]/,
+						name( module ) {
+							const moduleFileName = module
+								.identifier()
+								.split( '/' )
+								.reduceRight( ( item ) => item );
+							return `vendor/${ moduleFileName }`;
+						},
+						automaticNameDelimiter: '--',
+						minSize: 20000,
+						priority: -20,
+					},
```

Comment:
This moves all shared vendor files into single output, things like Dinero and such.